### PR TITLE
#348 : Adding JWT section

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: training/040_projects_and_infrastructure/020_Beginner.rst [hunk 1]
```diff
@@ -187,13 +187,23 @@ clean code
 JWT/Authentisering
 ~~~~~~~~~~~~~~~~~~
 
-Hvordan fungerer JWT’er?
+A JSON Web Token (JWT) is an open standard that is a compact and self-contained means of securely transmitting information between parties as a JSON object, i.e: A user and a Sesam node. JWTs, from a Sesam point of view, are used to allow a user to authenticate towards the Sesam Service API. This will in essence allow a user to communicate with a Sesam node without having to authenticate towards a Sesam node in your preferred browser. Everything from scripts and/or CLIs can now be used to communicate with your Sesam node.
```
Comment (ga-hegsvold) [inline training/040_projects_and_infrastructure/020_Beginner.rst:190]: I think it's important to distinguish between authentication and authorization here:
_Authentication_ is the process of verifying that a party is who they claim to be.
_Authorization_ is the process of verifying the access rights of a party.

As far as I know a user must first be _authenticated_ by logging in to a Sesam node before he/she can generate a JWT.
Then the JWT can be used as _authorization_ to access different parts of the node.

So JWTs are not used for authentication, but for authorization after authentication have been confirmed.

Thoughts?

File: training/040_projects_and_infrastructure/020_Beginner.rst [hunk 2]
```diff
@@ -187,13 +187,23 @@ clean code
 JWT/Authentisering
```
Comment (ga-hegsvold) [inline training/040_projects_and_infrastructure/020_Beginner.rst:187]: Funny title ;-) Maybe just remove "Authentisering".

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]

File: training/040_projects_and_infrastructure/020_Beginner.rst (hunk 1 above)
Comment (JonasAls89) [reply training/040_projects_and_infrastructure/020_Beginner.rst:190]: good point GAH. Will change accordingly.

Comment (JonasAls89) [reply training/040_projects_and_infrastructure/020_Beginner.rst]: Implemented

File: training/040_projects_and_infrastructure/020_Beginner.rst [hunk 3]
```diff
@@ -182,18 +182,28 @@ clean code
...
 
-NB: Skal snake mer om API I sesam-in-the-wild
+   Sesam JWT
 
 .. seealso::
 
-  TODO
+  Sesam in the Wild: :ref:`sesam-in-the-wild-beginner-6-1`
+
+  Developer Guide > Service API :ref:`api-top`
```
Comment (ga-hegsvold) [inline training/040_projects_and_infrastructure/020_Beginner.rst:204]: Can you add ":" before the ref (same as for "Sesam in the Wild" above)?
The title of the referred section is "Service API" so you can skip hard coding it. Shows up twice now.

File: training/040_projects_and_infrastructure/020_Beginner.rst [hunk 4]
```diff
@@ -182,18 +182,28 @@ clean code
...
+
+  Tools > Databrowser :ref:`databrowser_ini_authorization_via_jwt_token`
```
Comment (ga-hegsvold) [inline training/040_projects_and_infrastructure/020_Beginner.rst:206]: Can you add ":" before the ref?

**Review by ga-hegsvold — CHANGES_REQUESTED** [review]
I think this is good. Just a couple of tiny adjustments under `.. seealso`.

**Review by ga-hegsvold — APPROVED** [review]

**Pull request merged**